ui/src/message_popup/experience_page.jsx

```diff
@@ -212,11 +214,47 @@ export default React.createClass({
   },
   
   renderMobilePrototype() {
-    return (
-      <MobilePrototypeCard 
-        question={_.shuffle(withStudents(this.state.questions))[0]}
+    const {
+      questions,
+      questionsAnswered,
+      shouldShowStudentCard,
+      shouldShowSummary,
+      helpType,
+      sessionLength
+    } = this.state;
+    const question = questions[questionsAnswered];
+    return ( 
+      <MobileInterface
+        hasStarted = {this.state.hasStarted}
```

Comment:
As you were saying in person, it'd be good to land the refactoring in https://github.com/mit-teaching-systems-lab/threeflows/pull/67 first so that this can become a simpler interface too.